src/me/components/Resources.tsx

```diff
@@ -34,32 +33,18 @@ class ResourceLists extends PureComponent<Props> {
         stretchToFitWidth={true}
         margin={ComponentSize.Small}
       >
-        <Panel>
-          <Panel.Header>
-            <Heading
-              element={HeadingElement.H2}
-              weight={FontWeight.Light}
-              className="cf-heading__h4"
-            >
-              Account
-            </Heading>
-            <LogoutButton />
-          </Panel.Header>
-        </Panel>
         <Panel testID="recent-dashboards--panel">
           <Panel.Header>
             <Heading
               element={HeadingElement.H2}
-              weight={FontWeight.Light}
+              weight={FontWeight.Medium}
               className="cf-heading__h4"
             >
-              <label htmlFor="filter-dashboards">Recent Dashboards</label>
+              <label htmlFor="documentation-search">Documentation</label>
             </Heading>
           </Panel.Header>
           <Panel.Body>
-            <GetResources resources={[ResourceType.Dashboards]}>
-              <DashboardsList />
-            </GetResources>
```

Comment:
oh interesting, we're removing the dashboard list?

Reply:
yes